nemo_skills/inference/model/vllm.py

```
@@ -28,6 +28,10 @@ class VLLMModel(BaseModel):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
+    def _get_api_key(self, api_key: str | None, api_key_env_var: str | None, base_url: str) -> str | None:
```

Ruff (0.13.3)

Unused method argument: `api_key`

(ARG002)